File: TourMateBE/Repositories/Models/TourBidComment.cs
using System;
using System.Collections.Generic;

namespace Repositories.Models;

public partial class TourBidComment
{
    public int CommentId { get; set; }

    public int AccountId { get; set; }

    public int TourBidId { get; set; }

    public string Content { get; set; } = null!;

    public DateTime CreatedAt { get; set; }

    public bool IsDeleted { get; set; }

    public virtual Account Account { get; set; } = null!;

    public virtual TourBid TourBid { get; set; } = null!;
}

File: tourmatefe/types/tour-bid-comment.ts
export type TourBidComment = {
    commentId: number,
    accountId: number,
    tourBidId: number,
    content: string,
    createdAt: string,
    isDeleted: boolean
}

export type TourBidCommentCreateModel = {
    accountId: number,
    tourBidId: number,
    content: string,
}

export type TourBidCommentListResult = {
    commentId: number,
    accountId: number,
    tourBidId: number,
    content: string,
    createdAt: string,
    image: string,
    fullName: string
}